src/network-manager/azext_network_manager/_params.py

```diff
@@ -536,3 +542,150 @@ def load_arguments(self, _):
         c.argument('flag', type=str, help='Default rule flag.')
         c.argument('kind', type=str, help='Required. Whether the rule is custom or default.Constant filled by server.', arg_type=get_enum_type(['Custom', 'Default']))
         c.ignore('user_rule')
+
+    with self.argument_context('network manager connection create') as c:
+        c.argument('resource_group_name', resource_group_name_type)
+        c.argument('network_manager_connection_name', options_list=['--name', '-n', '--network-manager-connection-name'],
+                   type=str, help='The name of the network manager connection.')
+        c.argument('network_manager_id', type=str, help='the name or id of the network manager.', validator=validate_network_manager)
```

Comment:
please use '--network-manager' instead of '--network-manager-id'
```suggestion
        c.argument('network_manager_id', options=['--network-manager'], type=str, help='the name or id of the network manager.', validator=validate_network_manager)
```